### PR TITLE
Zeplin integration

### PR DESCRIPTION
**Pull request by TalZa1810**
### Summary
Integration of Zeplin with our library. 

## Why?
Help our users to better understand which components to use and by that improving the development process using `wix-style-react`. 

## Cons on Zeplin integration 
- Does not work for components that are exposed from `wix-ui-backoffice`.
- Currently, there's no option to map a UX story to multiple components ( in case of components composition). 
- Expose just the propTypes and not the prop values.

### CODE REVIEW

**Comment by shlomitc** [comment]
nice research but I think it doesn't give enough value at the moment

File: .zeplin/components.json [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+            "name": "4.4 Radio",
+            "path": "src/RadioGroup/RadioGroup.js",
+            "zeplinNames": [
+                " 04 Selections / 4.4 Radio / Group",
+                "04 Selections / 4.4 Radio / Single"
+            ]
+        },
+        {
+            "name": "4.5 Toggle",
+            "path": "src/ToggleButton/ToggleButton.js",
```
Comment (milkyfruit) [inline .zeplin/components.json]: this should be a `<ToggleSwitch/>`

**Comment by milkyfruit** [comment]
@shlomitc @TalZa1810 I want to reopen the discussion about Zeplin – Storybook integration.
I think we can find a workaround how to solve con # 2:

> Currently, there's no option to map a UX story to multiple components ( in case of components composition).

The integrations allows to add [repository links](https://github.com/zeplin/connected-components-docs/blob/master/docs/link/REPOSITORY.md), [custom links](https://github.com/zeplin/connected-components-docs/blob/master/docs/link/CUSTOM.md) and even [Storybook links](https://github.com/zeplin/cli-connect-storybook-plugin/).

If I understood the documentation correctly, it allows to add multiple links to stories manually. This way we could connect `<FormField/>` and `<Input/>` components to a single symbol:

```
{
    ...
    "components": [
        {
            "path": "inputPathName.jsx",
            "zeplinNames": [
                "03 Inputs / 3.1 Text Input / Large",
                "03 Inputs / 3.1 Text Input / Medium / 2 Label Start",
                "03 Inputs / 3.1 Text Input / Medium / 2 Label Top (Default)",
                "03 Inputs / 3.1 Text Input / Small"
            ],
            "storybook": {
                "kind": "kindName",
                "stories": [
                    FormField",
                    "Input",
                    "InfoIcon"
                ]
            }
        }
    ]
    ...
}
```

Also, the graphic example and [this article](https://blog.zeplin.io/introducing-connected-components-components-in-design-and-code-in-harmony-aa894ed5bd95
) tells it's possible to add a description, but unfortunately, I couldn't find a code example how to do it:
image

**Comment by shlomitc** [comment]
@TalZa1810 - can you try this one again following Domas's comment?

**Comment by TalZa1810** [comment]
@shlomitc yes. I'll re-check this.

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_4701/

**Comment by TalZa1810** [comment]
We should try this on a real project to check the usability. This can be part of a guid week. Since this currently not in priority, I'm closing this one.

**Pull request closed without merging**